src/scores/categorical/multicategorical_impl.py

```diff
@@ -18,9 +19,10 @@ def firm(  # pylint: disable=too-many-arguments
     categorical_thresholds: Sequence[float],
     threshold_weights: Sequence[Union[float, xr.DataArray]],
     discount_distance: Optional[float] = 0,
-    reduce_dims: Optional[Sequence[str]] = None,
-    preserve_dims: Optional[Sequence[str]] = None,
+    reduce_dims: FlexibleDimensionTypes = None,
+    preserve_dims: FlexibleDimensionTypes = None,
     weights: Optional[xr.DataArray] = None,
+    threshold_alignment: Optional[str] = "right",
```

Comment:
Did you already discuss choice of the term 'threshold_alignment'? If so, leave it. If not we could consider an alternative. E.g. threshold_assignment with options of "lower" and "upper" referring to whether a forecast or obs equal to the threshold is assigned to the lower or upper category. This might be more intuitive than left and right closed intervals.

Reply:
updated